<launch>
  <node pkg="rosbag" type="play" name="player"  args="--clock -l /home/jesus/rosbags/hand-90deg.bag"/>
 <node pkg="color_segmentation" type="sat_segmentation_node" name="sat_segmentation_node" output="screen">
   <remap from="/image" to="/hsrb/hand_camera/image_raw" />
   <!-- <remap from="/image" to="/hsrb/hand_camera/image_raw" /> -->
   <param name="minSat" value="150"/>
   <param name="maxSat" value="255"/>
   <param name="debug" value="true"/>
   <param name="y_center" value="180"/>
   <param name="x_center" value="120"/>
   </node>
</launch>
